client/src/modules/dashboard/Events/components/SendEmailModal.tsx

```diff
@@ -99,10 +99,15 @@ const SendEmailModal: React.FC<SendEmailModalProps> = ({
         </ModalBody>
 
         <ModalFooter>
-          <Button colorScheme="teal" mr={3} onClick={onClose}>
+          <Button mr={3} onClick={onClose}>
             Close
           </Button>
-          <Button type="submit" variant="solid" form="sendemail">
+          <Button
+            type="submit"
+            colorScheme={'green'}
```

Comment:
```suggestion
            colorScheme={'accent'}
```

Reply:
you don't have to create a theme, just change it and I will create a quick pull request with colors themes for style guide, too